README.md

```diff
@@ -50,3 +92,12 @@ Licensed under the GPLv2
     You should have received a copy of the GNU General Public License
     along with this program; if not, write to the Free Software
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+See the [COPYING][copying] file for the full license information.
```

Comment:
In that case the same applies for [libratbag](https://github.com/libratbag/libratbag#license) ;)